docs/reference/commandline/attach.md

```diff
@@ -34,6 +34,11 @@ Use `docker attach` to attach your terminal's standard input, output, and error
 ID or name. This allows you to view its ongoing output or to control it
 interactively, as though the commands were running directly in your terminal.
 
+> **Note:**
+> The attach command will display the output of the ENTRYPOINT/CMD process.  This
```

Comment:
Please mark this up a bit by putting technical words like "attach" and "ENTRYPOINT/CMD" in back-ticks.

Only a single space between sentences.

Other than that, this is great!